climpred/metrics.py

```diff
@@ -316,12 +478,14 @@ def _ppp(forecast, reference, dim='svd', comparison=None):
     """
     mse_skill = _mse(forecast, reference, dim=dim)
     var = reference.std(dim)
+    if 'comparison' in kwargs:
+        comparison = kwargs['comparison']
     fac = _get_norm_factor(comparison)
```

Comment:
Should there be an else above that throws an error? It looks like comparison is required here.